package.json

```diff
@@ -42,6 +42,7 @@
     "eslint-config-prettier": "^8.0.0",
     "eslint-plugin-import": "^2.18.0",
     "expect.js": "^0.3.1",
+    "jest": "^26.6.3",
     "mocha": "^8.0.1",
```

Comment:
Can we remove mocha and expect.js as well?

Reply:
done, and the nyc is removed as well